### ISSUE TITLE
The "Open file" and "Show file in folder" buttons do not work

### ISSUE THREAD

**Issue by rrondo** [StefanLobbenmeier/youtube-dl-gui#46]
Hi! For some reason, buttons such "Open file" and "Show file in folder" not working for me. Open file always end up with error, and Show file in folder just doing nothing.

Folder location – C:\Users\username\Downloads\YTDL

Neither the file name nor the folder name (and path to this folder) contains spaces in the name. The file name contains only Latin letters, no special characters (such "asdmovie")

 - OS: Windows 10
 - Application version 2.4.8 
 - Application type: installed

https://user-images.githubusercontent.com/46533574/231494564-65a28dfc-265b-474a-bb30-094a12d241c4.mp4

Thanks.

**Comment by ENTPRESTIGIOUS** [comment]
Same as https://github.com/StefanLobbenmeier/youtube-dl-gui/issues/44. This seems to have existed when jely was maintaining I believe.

**Comment by StefanLobbenmeier** [comment]
Yeah it’s one of the many annoyances in this version. I don’t want to fix them all individually and instead have a bigger overhaul. In this case the solution is to ask yt-dlp for the Filename which works quite nicely

**Comment by rrondo** [comment]
In 2.4.9 problem solved. Both buttons work fine now. Thank you, Stefan!

**Comment by StefanLobbenmeier** [comment]
oh nice, I think this was then caused by releasing #22

**Comment by ENTPRESTIGIOUS** [comment]
This error still persists when there are unrecognized ASCII characters like the ":" as it was represented by an underscore "_"

**Comment by ENTPRESTIGIOUS** [comment]
Opened with https://github.com/StefanLobbenmeier/youtube-dl-gui/issues/53
